sim/rogue/talents.go

```diff
@@ -346,7 +346,7 @@ func (rogue *Rogue) applyWeaponSpecializations() {
 			})
 		case proto.WeaponType_WeaponTypeMace:
 			rogue.OnSpellRegistered(func(spell *core.Spell) {
-				if spell.ProcMask.Matches(core.ProcMaskMeleeMH) {
+				if spell.ProcMask.Matches(core.ProcMaskMeleeMH) || spell.ProcMask.Matches(core.ProcMaskMeleeOH) {
```

Comment:
Please use `rogue.AddStat(stats.ArmorPenetration, ...)` if a mace is equipped in either hand, so the actual amount of ArPen is displayed in the "Stats" panel. (The two "mace cases" here and below then simply set a "anyMaceEquipped" bool or so.)

Reply:
Also, when using maces in both MH and OH, the `OnSpellRegistered` handlers would wrongly add Mace Spec's `BonusArmorPenRating` twice.